pocs/state/states/default/pointing.py

```diff
@@ -104,5 +105,4 @@ def on_enter(event_data):
         pocs.next_state = 'tracking'
 
     except Exception as e:
-        pocs.say(
-            "Hmm, I had a problem checking the pointing error. Sending to parking. {}".format(e))
+        pocs.say("Hmm, I had a problem checking the pointing error. Sending to parking. {}", e)
```

Comment:
It feels like "sending to parking" is action at a distance given the next_state = parking way at the top.

Reply:
Sort of yes. All states (should) specify `parking` as the next state at the top of the file and only by proper action does it get changed to a different state. This was aiming to bring consistency.